New alien faction: the Bunrodea (#6438)
endless-sky/endless-sky: data/map planets.txt

	description `	As you observe more of the planet, you even managed to find evidence of intelligent life in the form of entire towns and cities of buildings, but they all seem long abandoned.`
	description `	Your ship's sensors detect high levels of ultraviolet radiation, and cursory scans of the atmosphere reveal a distinct lack of ozone or any other gas that would prevent the nearby star's UV light from reaching the surface.`

planet "Deli Kasi"
	attributes bunrodea farming
	landscape land/fields3
	description `Deli Kasi is a world of vast oceans and mountainous continents. At many locations across the planet, the tall mountain ranges trap moist air from the oceans, causing large amounts of rainfall in concentrated areas. This, combined with snow melt from the mountains, creates large fertile plains across much of the habitable land area, which has led to Deli Kasi becoming a major farming world.`

> 	spaceport `In addition to the usual services you would expect, the spaceport of Deli Kasi is also home to a number of genetic engineering labs geared toward the creation of more bountiful plant seeds. It's unsurprising that the Bunrodeans would genetically modify their food, but for as much as the Bunrodeans are capable of, nothing beats the usefulness of fertile farm land like what can be found on this planet.`

Reviewer: ```suggestion
	spaceport `In addition to the usual services you would expect, the spaceport of Deli Kasi is also home to a number of genetic engineering labs geared toward the creation of more bountiful plant seeds. It's unsurprising that the Bunrodeans would genetically modify their food, but for as much as the Bunrodeans are capable of, nothing beats the usefulness of fertile farmland like what can be found on this planet.`
```

	spaceport `The station's spaceport is extremely large compared to the size of the entire station itself, taking up almost half of the interior. A variety of ships enter and exit, but a good chunk of the parked ships seem to be kept here and fitted for combat.`
	spaceport `	You're able to use some basic services in the spaceport, and there's even a small outfitter you can access, but you're prevented from entering the station beyond the spaceport.`
	outfitter "Bunrodea Basic"
	"required reputation" 50
	bribe 0
	security 0

planet "Illbo Avo"
	attributes bunrodea military moon
	landscape land/badlands6
	description `Illbo Avo is a barren moon with little to no atmosphere, but it is also the only solid ground in this system that isn't prone to sliding out under your feet as a result of volcanism. As such, because of the system's proximity to Paru Paru and to the Korath, the Bunrodea have turned the crust of this small moon into a dense network of subterranean structures with purposes ranging from large ship hangars to factories that pump out military-grade ships and outfits, essentially making it one big war factory.`

> 	spaceport `The main spaceport of Illbo Avo is accessed by entering a large artificial circular cavern near the moon's equator. The walls of the cavern are lined with hangar bays, with available bays having their hangar doors open. Once you find a vacant bay to land in, the hangar door closes and the hangar becomes pressurized.`

Reviewer: ```suggestion
	spaceport `The main spaceport of Illbo Avo is accessed by entering a large, artificial, circular cavern near the moon's equator. The walls of the cavern are lined with hangar bays; available bays have their hangar doors open. Once you find a vacant bay to land in, the door closes and the hangar becomes pressurized.`
```

	spaceport `	The inside of the spaceport is buzzing with activity, from shipments of new weapons being moved around to military squads marching in unison.`
	shipyard "Bunrodea Basic"
	shipyard "Bunrodea Advanced"
	outfitter "Bunrodea Basic"
	outfitter "Bunrodea Advanced"
	"required reputation" 25
	bribe 0
	security 0

planet "Illbo Elo"
	attributes bunrodea military
	landscape land/badlands0
	description `Illbo Elo is a sparsely populated world, with much of the planet's surface being barely habitable deserts. The climate becomes more temperate as you approach the poles, where most of the population is located. The Bunrodea appear to be constructing entirely new cities within these temperate zones, as evident by the large fleets of freighters bringing supplies to cities of mostly skeletal buildings with exposed foundations and metal beams.`

> 	spaceport `The main spaceport of Illbo Elo appears to be a major military installation, with most of the landing pad being occupied by Bunrodean warships. Moving away from the more military focused center, you find that the edges of the spaceport are packed with civilians barely supported by the current infrastructure, which is perhaps why the Bunrodea are constructing new cities elsewhere.`

Reviewer: ```suggestion
	spaceport `The main spaceport of Illbo Elo appears to be a major military installation, with most of the landing pad being occupied by Bunrodean warships. Moving away from the more military-focused center, you find that the edges of the spaceport are packed with civilians barely supported by the current infrastructure, which is perhaps why the Bunrodea are constructing new cities elsewhere.`
```

> 	spaceport `The main spaceport of Illbo Elo appears to be a major military installation, which most of the landing pad being occupied by Bunrodean warships. Moving away from the more military focused center, you find that the edges of the spaceport are packed with civilians, with the infrastructure barely supporting them, which is perhaps why the Bunrodea are constructing new cities elsewhere.`

Reviewer: ```suggestion
	spaceport `The main spaceport of Illbo Elo appears to be a major military installation, with most of the landing pad being occupied by Bunrodean warships. Moving away from the more military focused center, you find that the edges of the spaceport are packed with civilians barely supported by the current infrastructure, which is perhaps why the Bunrodea are constructing new cities elsewhere.`
```

	outfitter "Bunrodea Basic"
	outfitter "Bunrodea Advanced"
	"required reputation" 25
	bribe 0
	security 0

planet "Disara"
	attributes bunrodea urban military
	landscape land/forest2

> 	description `Disara is a highly developed world, with multiple large cities on each of the planet's continents. Each of the cities spreads out into more surban-like regions, which extend further into rural and wilderness zones. Unlikely many other worlds, Disara has no one primary industry that it dominates in. Instead, the planet does a bit of everything, having a rather strong internal economy.`

Reviewer: ```suggestion
	description `Disara is a highly developed world, with multiple large cities on each of the planet's continents. Each of the cities spread out into more suburban-like regions, which extend further into rural and wilderness zones. Unlikely many other worlds, Disara has no one primary industry that it dominates in. Instead, the planet does a bit of everything, having a rather strong internal economy.`
```

> 	spaceport `Part of the outskirts of the planet's main spaceport has been converted into a military installation, which a large base garrisoned with a considerable number of royal guards and their many ships. Apart from that, the spaceport feels rather normal, at least by Bunrodean standards. You spot the occasional Bunrodean with an excessive number of cybernetic or genetic modification, but most appear to have few, if any, modifications done to them.`

Reviewer: ```suggestion
	spaceport `Part of the outskirts of the planet's main spaceport has been converted into a military installation, which a large base garrisoned with a considerable number of royal guards and their many ships. Apart from that, the spaceport feels rather normal, at least by Bunrodean standards. You spot the occasional Bunrodean with an excessive number of cybernetic or genetic modifications, but most appear to have few, if any, alterations done to them.`
```

	shipyard "Bunrodea Basic"
	shipyard "Bunrodea Advanced"
	outfitter "Bunrodea Basic"
	outfitter "Bunrodea Advanced"
	"required reputation" 25
	bribe 0

planet Jentuthro
	attributes bunrodea guild medical
	landscape land/sky5

> 	description `Jentuthro orbits one of the dimmest stars in this region of space, making it an unusual place to find the Bunrodea. It is also where you will find their major genetic engineering research facilities. These facilities attract the brightest minds in genetic engineering from across all of Bunrodean space, where they work to further their understand and control over genetics.`

Reviewer: ```suggestion
	description `Jentuthro orbits one of the dimmest stars in this region of space, making it an unusual place to find the Bunrodea. It is also where you will find their major genetic engineering research facilities. These facilities attract the brightest minds in genetic engineering from across all of Bunrodean space, where they work to further their understanding and control over genetics.`
```

	bribe 0
	security 0

planet Kalbai
	attributes bunrodea uninhabited
	landscape land/forest4
	description `For some reason, this apparently fruitful world has been left largely unsettled by the Bunrodea. Kalbai is teeming with life of all kinds, from varieties of plants across the various forests and plains of the planet's surface to the critters of varying sizes that scurry through the underbrush.`
	description `	There are only minimal signs that anyone has ever set foot on this world; there are a number of ancient ruins scattered across the largest continent. A few appear to have been abandoned and reclaimed by nature, but most look as though they were destroyed, creating strikingly lifeless circles in the landscape around the center of the ruins.`

planet Kisarra
	attributes bunrodea mining
	landscape land/desert2
	description `Kisarra's surface is one vast desert as far as the eye can see. The sandy nothingness is only occasionally broken up by oases and rocky outcrops. Many of the oases have small towns that have formed around them and some of the outcrops seem to used as entrances to mines. Strikingly, all of the towns and mines have large walls constructed around them.`

> 	spaceport `Although most spaceports typically double as a massive population center for the planet, in many cases being the most populous region on the planet, Kisarra's spaceport acts only as a location to ferry minearls brought from the mines off-world, with most of the planet's population living in the small oasis towns.`

Reviewer: ```suggestion
	spaceport `Although most spaceports typically double as a massive population center for the planet, in many cases being the most populous region on the planet, Kisarra's spaceport acts only as a location to ferry minerals brought from the mines off-world, with most of the planet's population living in the small oasis towns.`
```

> 	spaceport `	Just as with the small towns and mines, the spaceport has a large wall constructed around it. From what you can tell, this is to keep out the gigantic hostile spider-like creatures that inhabit Kissara's deserts. These spiders burrow through the sand and consume any unfortunate creature they can find.`

Reviewer: ```suggestion
	spaceport `	Just as with the small towns and mines, the spaceport has a large wall constructed around it. From what you can tell, this is to keep out the gigantic spider-like creatures that inhabit Kissara's deserts. These spiders burrow through the sand and consume any unfortunate creature they can find.`
```

	spaceport `The spaceport of Melenci has a variety of what you would assume to be advertisements for genetic operations. One displays a graphic of two animated Bunrodeans in a mine. One is injected with a needle and then become much bulkier. The mine then collapses in on the two, and the Bunrodean who was genetically modified is shown coming out of the rubble unscathed, while the other is pulled out with bent limbs and placed on a stretcher.`
	shipyard "Bunrodea Basic"
	outfitter "Bunrodea Basic"
	"required reputation" 50
	bribe 0
	security 0

planet Okoity
	attributes bunrodea urban
	landscape land/fog2
	description `Okoity has the highest population of any Bunrodean world, with a staggering 24 billion people. All but the most inhospitable regions of the planet have been settled in some shape or form.`
	description `	The planet saw a large influx of people with the destruction of Olohoa's atmosphere in the nearby system of Sera Natta, massive even relative to the already preexisting population. Although this occurred about a century ago, the planet still seems to be struggling with this increased population.`

> 	spaceport `The spaceport of Okoity is completely packed with Bunrodeans. You have little room to maneuver as you explore the spaceport while trying to not bump into anyone. Many of the Bunrodeans here show the signs of being genetically engineered, from Bunrodeans with extra limbs to one that you see up high climbing on walls and jumping from building to building to avoid the crowd below.`

Reviewer: ```suggestion
	spaceport `The spaceport of Okoity is completely packed with Bunrodeans. You have little room to maneuver as you explore the spaceport while trying to not bump into anyone. Many of the Bunrodeans here show signs of being genetically engineered, from Bunrodeans with extra limbs to one that you see up high climbing on walls and jumping from building to building to avoid the crowd below.`
```

	shipyard "Bunrodea Basic"
	outfitter "Bunrodea Basic"
	"required reputation" 25
	bribe 0
	security 0

planet Olohoa
	attributes bunrodea uninhabited
	landscape land/sky3
	description `From afar, Olohoa looks like it might be a pleasant place to live, with clear oceans not too dissimilar to what you might find on a Paradise World. But your ship's sensors tell you a different story, one of a planet with immense amounts of ultraviolet radiation bombarding its surface. Nothing short of an extremophile could survive on the surface here, and any life in the oceans is likely deep down where no radiation can reach.`
	description `	As you explore, you begin to find occasional signs of civilization in the form of small cities with landing pads for ships, but these areas are just as lifeless as the rest of the planet.`

planet Sardva
	attributes bunrodea religious
	landscape land/fields6

> 	description `The planet of Sardva is used by the Bunrodea as a vault for genetic material of all origins, from the seeds of plants that the Bunrodea have encountered to the genetic material of animals and even past generations of Bunrodeans. This material is stored in many of the planet's cave systems, which, with some slight modification, have amazingly stable climates that are perfect for the long term storage of genetic material.`

Reviewer: ```suggestion
	description `The planet of Sardva is used by the Bunrodea as a vault for genetic material of all origins, from the seeds of plants that the Bunrodea have encountered to the genetic material of animals and even past generations of Bunrodeans. This material is stored in many of the planet's cave systems, which, with some slight modification, have amazingly stable climates that are perfect for the long term storage of genetic libraries.`
```

> 	description `The planet of Sardva is used by the Bunrodea as a vault for genetic material of all origins, from the seeds of plants that the Bunrodea have encountered to genetic material of animals and even material from past generations of Bunrodeans. This material is stored in many of the planet's cave systems, which, with some slight modification, have amazingly stable climates that are perfect for the long term storage of genetic material.`

Reviewer: ```suggestion
	description `The planet of Sardva is used by the Bunrodea as a vault for genetic material of all origins, from the seeds of plants that the Bunrodea have encountered to the genetic material of animals and even past generations of Bunrodeans. This material is stored in many of the planet's cave systems, which, with some slight modification, have amazingly stable climates that are perfect for the long term storage of genetic material.`
```

	description `	The decentralized and redundant nature of the vaults also ensures that if anything happens on one part of the planet that may compromise the vaults there, locations elsewhere on the planet will still keep the material safe.`
	spaceport `Sardva has no single major spaceport, with each vault location supporting its own small spaceport for receiving shipments and refueling the freighters. You eventually find a vault that is willing to let you land so that you may refuel, but they don't seem too happy about your presence, and seem like they're only refueling you so that you stop asking every vault you come across.`
	"required reputation" 200
	bribe 0
	security 0

planet Schisathro
	attributes bunrodea guild manufacturing mining
	landscape land/canyon5
	description `Schisathro is a major mining world, and also home to some of the largest shipyards in Bunrodean space. Larger shipyards than the ones found here only exist on the capital world of Erabuthro.`

> 	description `	The planet's success as a mining world can be owed to its many deep canyons found across most of the planet's above water surface, which allow for easier access to mineral rich deposits in the crust. The major shipyards developed alongside these mines allowed for easy transportation of material directly to the shipyards instead of requiring lengthy journeys through space.`

Reviewer: ```suggestion
	description `	The planet's success as a mining world can be owed to its many deep canyons found across most of the planet's above water surface, which allow for easier access to mineral-rich deposits in the crust. The major shipyards developed alongside these mines allowed for easy transportation of material directly to the shipyards instead of requiring lengthy journeys through space.`
```

	shipyard "Bunrodea Basic"
	outfitter "Bunrodea Basic"
	"required reputation" 10
	bribe 0
	security 0

planet Thshybothro
	attributes bunrodea guild medical
	landscape land/mountain0
	description `Thshybothro is a small, mountainous world that is home to the Bunrodea's largest cybernetics research facilities. Various labs and factories are scattered within the world's many valleys, within which the Bunrodea work to develop new cybernetics, or improve their old ones.`
	description `	Although this planet is where new modifications are developed, the mass production and installation of cybernetics is typically done elsewhere.`
	spaceport `The planet's spaceport is rather small, as typically any visitors to the planet travel directly to the research facilities that they're interested in. As such, the spaceport's primary focus is simply to refuel any ships that stop by the planet and who don't have authorization to land at one of the research facilities.`

> 	spaceport `	The spaceport dock workers have more cybernetic enhancements than you're used to from Bunrodeans elsewhere in their space. The worker who refuels your ship in particular is a mass of cybernetic "enhancements" ranging from powerful legs to aid in lifting crates to a vacuum cleaner attachment for dealing with spills and accidents.`

Reviewer: ```suggestion
	spaceport `	The spaceport dock workers have more cybernetic modifications than you're used to from Bunrodeans elsewhere in their space. The worker who refuels your ship in particular is a mass of cybernetic "enhancements" ranging from powerful legs to aid in lifting crates to a vacuum cleaner attachment for dealing with spills and accidents.`
```

	bribe 0
	security 0

planet Kalbai
	attributes bunrodea uninhabited
	landscape land/forest4
	description `For some reason, this apparently fruitful world has been left largely unsettled by the Bunrodea. Kalbai is teeming with life of all kinds, from varieties of plants across the various forests and plains of the planet's surface to the critters of varying sizes that scury through the underbrush.`
	description `	There are only minimal signs that anyone has ever set foot on this world; there are a number of ancient ruins scattered across the largest continent. A few appear to have been abandoned and reclaimed by nature, but most look as though they were destroyed, creating strikingly lifeless circles in the landscape around the center of the ruins.`

planet Kisarra
	attributes bunrodea mining
	landscape land/desert2
	description `Kisarra's surface is one vast desert as far as the eye can see. The sandy nothingness is only occasionally broken up by oases and rock outcrops. Many of the oases have small towns that have formed around them and some of the rock outcrops seem to used as entrances to mines. Strikingly, all of the towns and mines have large walls constructed around them.`

> 	spaceport `Although most spaceports typically double as a massive population center for the planet, in many cases being the most populous region on the planet, Kisarra's spaceport acts only as a location to ferry minearls brought from the mines off planet, with most of the planet's population living in the small oasis towns.`

Author: The intention here was actually to use a phrase synonymous with "off-world."

	spaceport `Sardva has no single major spaceport, with each vault location supporting its own small spaceport for receiving shipments and refueling the freighters. You eventually find a vault that is willing to let you land so that you may refuel, but they don't seem too happy about your presence, and seem like they're only refueling you so that you stop asking every vault you come across.`
	"required reputation" 200
	bribe 0
	security 0

planet Schisathro
	attributes bunrodea guild manufacturing mining
	landscape land/canyon5
	description `Schisathro is a major mining world, and also home to some of the largest shipyards in Bunrodean space. Larger shipyards than the ones found here only exist on the capital world of Erabuthro.`
	description `	The planet's success as a mining world can be owed to its many deep canyons found across most of the planet's above water surface, which allow for easier access to mineral rich deposits in the crust. The major shipyards developed alongside these mines allowed for easy transportation of material directly to the shipyards instead of requiring lengthy journeys through space.`

> 	spaceport `The main spaceport of Schisathro is almost entirely taken up by shipyards. That area which isn't dedicated to the shipyards is dedicated to the transportion of materials to them. This leaves very little space left over for the usual amenities that you would expect of a spaceport. There is only one single small outfitter tucked in between two large shipyards near the edge of the spacport; the shipyards must receive direct shipments of the more advanced outfits that they need.`

Reviewer 2: ```suggestion
	spaceport `The main spaceport of Schisathro is almost entirely taken up by shipyards. That area which isn't dedicated to the shipyards is dedicated to the transportation of materials to them. This leaves very little space left over for the usual amenities that you would expect of a spaceport. There is only one single small outfitter tucked in between two large shipyards near the edge of the spaceport; the shipyards must receive direct shipments of the more advanced outfits that they need.`
```

> 	spaceport `The main spaceport of Schisathro is almost entirely taken up by shipyards. That area which isn't dedicated to the shipyards is dedicated to the transportion of materials to them. This leaves very little space left over for the usual amenities that you would expect of a spaceport. There is only one single small outfitter tucked in between two large shipyards near the edge of the spacport; the shipyards must receive direct shipments of the more advanced outfits that they need.`

Reviewer 2: Not a huge fan of the repeated 'transportation of materials'.

Author: They're at least on separate screens, since one is the planet description and the other is the spaceport description.

	spaceport `Sopoyra is not a very hospitable world. There are few settlements outside of the main spaceport, which is almost entirely dedicated to factories and warehouses.`
	spaceport `	There are freighters that will come from the north to drop off goods here, only for them to be immediately picked up by another freighter that brings the goods down south. It seems somewhat inefficient to do things this way, but must be done to be in keeping with royal decrees.`
	shipyard "Bunrodea Basic"
	outfitter "Bunrodea Basic"
	"required reputation" 10
	bribe 0
	security 0

planet Thshybothro
	attributes bunrodea guild medical
	landscape land/mountain0
	description `Thshybothro is a small, mountainous world that is home to the Bunrodea's largest cybernetics research facilities. Various labs and factories are scattered within the world's many valleys, within which the Bunrodea work to develop new cybernetics, or improve their old ones.`
	description `	Although this planet is where new modifications are developed, the mass production and installation of cybernetics is typically done elsewhere.`

> 	spaceport `The planet's spaceport is rather small, as typically any visitors to the planet travel directly to the research facitilites that they're interested in. As such, the spaceport's primary focus is simply to refuel any ships that stop by the planet and who don't have authorization to land at one of the research facitlites.`

Reviewer 2: ```suggestion
	spaceport `The planet's spaceport is rather small, as typically any visitors to the planet travel directly to the research facilities that they're interested in. As such, the spaceport's primary focus is simply to refuel any ships that stop by the planet and who don't have authorization to land at one of the research facilities.`
```

> 	spaceport `	The spaceport dock workers have more cybernetic enhancements than you're used to from Bunrodeans elsewhere in their space. The worker who refuels your ship in particular is a mass of cybernetic "enhancements" ranging from powerful legs to aide in lifting crates to a vacuum cleaner attachment for dealing with spills and accidents.`

Reviewer 2: ```suggestion
	spaceport `	The spaceport dock workers have more cybernetic enhancements than you're used to from Bunrodeans elsewhere in their space. The worker who refuels your ship in particular is a mass of cybernetic "enhancements" ranging from powerful legs to aid in lifting crates to a vacuum cleaner attachment for dealing with spills and accidents.`
```

wormhole "Ember Wormhole"
	mappable

> 	link Aescolanus Cardea
> 	link Cardea Insitor
> 	link Insitor Segesta
> 	link Segesta Aescolanus
> 	color "wormholes: Ember Waste"

Author: ```suggestion
	color "wormholes: Ember Waste"
	link Aescolanus Cardea
	link Cardea Insitor
	link Insitor Segesta
	link Segesta Aescolanus
```

	spaceport `The spaceport is a sprawling complex of hangars and warehouses connected by covered walkways and underground tunnels. Because it has grown larger and larger over the millennia as this world's population increased, the spaceport's current layout is maze-like and seemingly random. Colored stripes painted on the floors mark the pathways between the major sections of the complex, and many local children earn a living just by wandering around offering to give directions to visitors who seem to be lost.`

planet Melenci
	attributes bunrodea medical moon
	landscape land/badlands8
	description `While the southern world of Jentu Le is where genetic research is mainly done, the small moon of Melenci here in the north is where much of that research is put to practice. The moon has only one settlement where Bunrodeans come to receive modifications to their genome. However, the Bunrodeans ensure that any modifications that are done will affect only the individual they are being done to, and will not be passed down to any offspring.`

> 	spaceport `The spaceport of Melenci has a variety of what you would assume to be advertisements for genetic operations. One displays a graphic of two animated Bunrodeans in a mine. One is injected with a needle and then become much bulkier. The mine then collapses in on the two, and the Bunrodean who was genetically modified is shown coming out of the rubble unscathed, while the other is pulled out with bent limbs and placed on a stretcher.`

Reviewer: ```suggestion
	spaceport `The spaceport of Melenci has a variety of what you would assume to be advertisements for genetic operations. One displays a graphic of two animated Bunrodeans in a mine. One is injected with a needle and then becomes much bulkier. The mine then collapses in on the two, and the Bunrodean who was genetically modified is shown coming out of the rubble unscathed, while the other is pulled out with bent limbs and placed on a stretcher.`
```
